web/src/routes/photos/+page.svelte

```diff
@@ -73,6 +73,28 @@
 		isShowAddMenu = !isShowAddMenu;
 	};
 
+	const handleAddToFavorites = () => {
+		isShowAddMenu = false;
+
+		var cnt = 0;
+		for (const asset of $selectedAssets) {
+			if (!asset.isFavorite) {
+				api.assetApi.updateAsset(asset.id, {
+					isFavorite: true
+				});
+				assetStore.updateAsset(asset.id, true);
+				cnt = cnt + 1;
+			}
+		}
```

Comment:
This works for now. We may want to refactor this into a single api request in the future (ie: bulk update via `PATCH /asset` or similar)